scripts/sql/87._ses_notification_template.down.sql

```diff
@@ -0,0 +1,66 @@
+---- update notification template for CI trigger ses/smtp
```

Comment:
File name is incorrect. ie `scripts/sql/87._ses_notification_template.down.sql` to `scripts/sql/87_ses_notification_template.down.sql`